### PR TITLE
Added "Test" build config

### LINKED ISSUE #87: Add /testing directory with appropriate Eclipse settings
Will need to make a folder thats wrapped in a #ifdef so it only executes during a specific configuration

### PR DESCRIPTION
**Pull request by tyrelkostyk**
See Issues #87 

Added the `Test` build configuration, and set the necessary project settings to make it all work. Also added a basic Test Suite framework; not sure how much it'll be used or if it'll always look like that, but it's one way of doing it.

**To test and approve this PR:**
1. Close Eclipse
2. Checkout the branch `admin/test-build-config`
3. Start Eclipse
4. Build the Test configuration (hit the arrow next to the hammer icon, and select `Test` instead of `Debug`)
5. There should be no build errors
6. Go to `main.c` -> Lines wrapped in `#ifdef TEST` should NOT be greyed out (e.g. lines 21-23 and 86-92)

### CODE REVIEW

**Comment by ChrisAmaya** [comment]
I followed your steps but both #ifdef TEST is grayed out. I tried to mess around with the settings and what worked for me is I went into the properties tab -> C/C++ General -> Indexer -> and changed the bottom drop-down to test. 

The problem I ran into while doing this was that I cannot change back to different builds using the dropdown menu by the build hammer icon I would have to change the configuration via that path mentioned before (C/C++ -> Indexer -> ...). 

I am unsure what they mean by "indexer" in the first place but it does seem to help with the configuration symbols

File: radsat-sk/src/main.c [hunk 1]
```diff
@@ -76,6 +80,25 @@ int main(void)
 	LED_wave(1);
 	LED_waveReverse(1);
 
+	// TODO: System Initialization (HAL, SSI, other low-level initializations, etc.)
+	// TODO: Subsystem Initialization (Transceiver, Downlink Manager, etc. if necessary)
+
+#ifdef TEST
+
+	// run unit test suite
+	testSuiteRunAll();
```
Comment (ChrisAmaya) [inline radsat-sk/src/main.c:89]: Cool abstract idea

**Review by ChrisAmaya — CHANGES_REQUESTED** [review]
Please review my comment I left about the settings I tried to get the configuration to work

**Comment by tyrelkostyk** [comment]
UPDATE: Fixed Addi's above issue (which I was also seeing)

1. Open Eclipse
2. Go to `Window > Preferences > C/C++ > Indexer`
3. On that page, near the middle/bottom, there will be an option: _Build Configuration for the Indexer_
    - Select **Use active build configuration**
4. Close Eclipse
5. Checkout the branch for this PR (`admin/test-build-config`) if not done already
6. Proceed to test & evaluate PR as explained in the initial PR comment above

**Review by ChrisAmaya — APPROVED** [review]

**Review by matthew-buglass — APPROVED** [review]
Looks good and works for me.

**Comment by tyrelkostyk** [comment]
Updated the Team Training doc as well, to include the necessary steps required to change the local project settings

Link to training doc: https://www.dropbox.com/home/CubeSat/Subteams/Software%20and%20Command/Learning/Training
^ **In section 4.5.4**

**Pull request merged**